llvm/lib/Target/SPIRV/SPIRVInstructionSelector.cpp

```diff
@@ -3305,6 +3395,24 @@ bool SPIRVInstructionSelector::selectSpvThreadId(Register ResVReg,
   return MIB.constrainAllUses(TII, TRI, RBI);
 }
 
+SPIRVType *
+SPIRVInstructionSelector::getCorrespondingVec4Type(const SPIRVType *Type,
```

Comment:
nit: name feels a bit weird, as if I could pass a `vec2<T>` and get a `vec4<vec2<T>>`
What about `splatToVec4` or `expantAsVec4` or `widenAsVec4`? Or something closer to SPIR-V naming: `composeVec4From`?

Reply:
I went with `widenTypeToVec4`.